### PR TITLE
Update to Play 2.5.x

### PR DESCRIPTION
**Pull request by wsargent**
Upgrades the Ebean Java sample to 2.5.1.

Note that this uses @adrianhurt play-bootstrap code https://github.com/adrianhurt/play-bootstrap as the Play template helper, as it looks much better than the custom field constructor.

### CODE REVIEW

File: test/FunctionalTest.java [hunk 1]
```diff
@@ -1,89 +1,113 @@
...
+    public void teardown() {
+        Helpers.stop(application);
+    }
+
     @Test
     public void redirectHomePage() {
-        running(fakeApplication(), new Runnable() {
-           public void run() {
-               Result result = callAction(controllers.routes.ref.Application.index());
+        Helpers.running(Helpers.fakeApplication(), () -> {
```
Comment (marcospereira) [inline test/FunctionalTest.java]: You won't need both `application` built at the `setup` method and this `fakeApplication` here. In other words, all these `Helpers.running` could be removed and the test be "plain".

**Comment by marcospereira** [comment]
Tests are broken. Also, maybe it could be a good idea to activate travis for these projects so PRs could be validated.

**Comment by wsargent** [comment]
```
[computer-database-java] $ test
[info] application - Creating Pool for datasource 'default'
[info] application - Shutting down connection pool.
[info] application - Creating Pool for datasource 'default'
[DEBUG] [04/13/2016 16:18:31.819] [application-akka.actor.default-dispatcher-4] [EventStream] shutting down: StandardOutLogger started
[info] application - Shutting down connection pool.
[info] application - Creating Pool for datasource 'default'
[info] application - Shutting down connection pool.
[info] Passed: Total 3, Failed 0, Errors 0, Passed 3
[success] Total time: 7 s, completed Apr 13, 2016 4:18:32 PM
```

Odd, tests are working for me...

**Comment by marcospereira** [comment]
Weird. To they are broken:

```
[error] Test FunctionalTest.createANewComputer failed: java.lang.IllegalStateException: Attempted to call materialize() after the ActorMaterializer has been shut down., took 0.189 sec
[error]     at akka.stream.impl.ActorMaterializerImpl.materialize(ActorMaterializerImpl.scala:88)
[error]     at akka.stream.impl.ActorMaterializerImpl.materialize(ActorMaterializerImpl.scala:79)
[error]     at akka.stream.scaladsl.RunnableGraph.run(Flow.scala:354)
[error]     at play.api.libs.streams.SinkAccumulator.run(Accumulator.scala:123)
[error]     at play.api.test.EssentialActionCaller$class.call(Helpers.scala:225)
[error]     at play.api.test.Helpers$.call(Helpers.scala:382)
[error]     at play.api.test.RouteInvokers$class.route(Helpers.scala:247)
[error]     at play.api.test.Helpers$.route(Helpers.scala:382)
[error]     at play.api.test.RouteInvokers$class.jRoute(Helpers.scala:234)
[error]     at play.api.test.Helpers$.jRoute(Helpers.scala:382)
[error]     at play.api.test.Helpers.jRoute(Helpers.scala)
[error]     at play.test.Helpers.route(Helpers.java:360)
[info] application - Shutting down connection pool.
[error]     at play.test.Helpers.route(Helpers.java:355)
[error]     at play.test.Helpers.route(Helpers.java:337)
[error]     at FunctionalTest.lambda$createANewComputer$3(FunctionalTest.java:79)
[error]     at play.test.Helpers.running(Helpers.java:386)
[error]     at FunctionalTest.createANewComputer(FunctionalTest.java:78)
```

Let's way for travis build.

**Pull request merged**